### ISSUE TITLE
User Refs as lists.

### ISSUE THREAD

**Issue by reality-exe** [Xlinka/NeosPlus#62]
(Based off of [this comment](https://github.com/Xlinka/NeosPlus/issues/57#issuecomment-1226696237) on issue #57)

Ability to use `+` & `-` nodes to group together `User References` to input into thing like `Ignore User` on certain nodes.

However on second though there might need to be some redoing of some of the nodes now that I think about it to make this compatible.

**Comment by DoubleStyx** [comment]
Yeah, the nodes would need to be patched to support collections of user refs. We could define our own nodes that accept those, but that would be a bit redundant. Also, we need to implement LogiX collections before this can be done.
